packages/alfa-rules/src/sia-er62/rule.ts

```diff
@@ -244,6 +244,11 @@ export default Rule.Atomic.of<Page, Element>({
                 )
               );
 
+              const sortedDistinguishableContrast = Array.sortWith(
+                [...distinguishableContrast],
+                (a, b) => a.compare(b)
+              );
+
```

Comment:
I would put the sorting directly in `ElementDistinguishrable.from` (and `ElementDistinguishable#withPairing`). This ensure the sorting is done all the time and notably we don't need to remember to sort the pairings in the tests.